src/components/LogOutButton.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Button } from 'react-native-elements';
+import { connect } from 'react-redux';
+import { NavigationActions } from 'react-navigation';
+import Block from './Block';
+import { logoutUser } from '../actions/UserActions';
+
+const LogOutButton = ({ logout }) => (
+  <Block width row ac js>
+    <Block mh={10}>
+      <Button
+        title="LogOut"
+        onPress={() => logout(NavigationActions.navigate)}
```

Comment:
` onPress={() => logout(NavigationActions.navigate)}`. Write arrow-functions in onPress, only, if it is impossible to avoid. 
Move logout(NavigationActions.navigate) to separate function